span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -242,7 +242,7 @@ private Event buildEvent(
 
     // Generate other fields if possible from the existing ones. eg. http scheme and query string
     // from url
-    this.fieldsGenerator.populateOtherFields(eventBuilder);
+    this.fieldsGenerator.populateOtherFields(eventBuilder, attributeFieldMap);
```

Comment:
if you are not changing `maybeSetHttpUrlForOtelFormat`, let's put a note here that attributeMap should contain all the spans tags before calling this method.

Reply:
sure